.travis.yml

```diff
@@ -1,14 +1,10 @@
----
-os: linux
-dist: bionic
-
-jobs:
-  include:
-    - language: python
-      python: 3.8
-      cache: pip
-      install: "pip install -r requirements.txt"
-      script: sh run_tests.sh
+language: python
+python:
+  - "3.8"
+# command to install dependencies
+install: "pip install $(pipenv lock --requirements)"
```

Comment:
How about using: `pipenv install` to ensure the Build works fine and then we can figure out a way to use pip instead ?

Reply:
Yes. @nasirhm. That seemed to have worked.